src/ex4/dist/main.js

```diff
@@ -0,0 +1,288 @@
+import ItemClient from './clients/item-client.js'
+
+export default class Main{
+    constructor(){
+        this.itemClient = new ItemClient()
+        this.todoList = []
```

Comment:
there is too much logic in this file. bring back the item-manager and move all todo related logic there.